src/main/webapp/app/therapeuticareas/views/therapeutic.area.questionnaire.view.htm

```diff
@@ -1,99 +1,33 @@
-                                </button>
-                              </span>
-                        </div>
-                    </div>
-
-
-                    <div class="answers" ng-if="questionnaireEntry.answers._embeddedItems.length<=4">
-                        <button class="left btn btn-remove" type="button" title="Delete"
-                                    data-ng-click="deleteAnswer(questionnaireEntry, answer)"  data-ng-repeat="answer in questionnaireEntry.answers._embeddedItems | orderBy:'id'">
-                            {{answer.answerText}}    <span class="glyphicon glyphicon-remove"></span>
-                        </button>
-
-
-
-                        <!--
-                        <div as-sortable="sortableOptions" ng-model="questionnaireEntry.answers._embeddedItems">
-                            <div ng-repeat="answer in questionnaireEntry.answers._embeddedItems" as-sortable-item class="as-sortable-item">
-                                <div as-sortable-item-handle>
-                                    <span data-ng-bind="answer.answerText"></span>
-                                </div>
-                            </div>
-                        </div>
-                        -->
-
-
-                    </div>
-
-                    <div class="answers" ng-if="questionnaireEntry.answers._embeddedItems.length>=5">
-                        <div class="option-list">
-                            <button class="left btn btn-remove" type="button" title="Delete"
-                                    data-ng-click="deleteAnswer(questionnaireEntry, answer)"  data-ng-repeat="answer in questionnaireEntry.answers._embeddedItems | orderBy:'id'">
-                                {{answer.answerText}}    <span class="glyphicon glyphicon-remove"></span>
-                            </button>
-                        </div>
-                    </div>
-                    <div class="clear"></div>
-
-
-                    
-
-                </div>
-            </div>
-        </div>
-
-
-        <!--
-        <div as-sortable="sortableOptions" ng-model="questionnaireEntries">
+    </div>
 
-        <div class="form-group" data-ng-repeat="questionnaireEntry in questionnaireEntries | orderBy:'-question.id'"  as-sortable-item class="as-sortable-item">
 
-            <div class="qa" as-sortable-item-handle class="as-sortable-item-handle">
+    <div class="form-group" data-as-sortable="sortableOptions" data-ng-model="questionnaireEntries">
+        <div data-ng-repeat="questionnaireEntry in questionnaireEntries" data-as-sortable-item class="as-sortable-item">
+            <div class="qa" data-as-sortable-item-handle class="as-sortable-item-handle">
 
                 <div class="questionEdit">
-                    <button class="left btn btn-remove btn-remove-question" type="button" title="Delete"
+                    <button id="btn_deleteQuestion" class="left btn btn-remove btn-remove-question" type="button" title="Delete"
```

Comment:
Can you suffix the questionnaireEntry.question.sortOrder to the btn_deleteQuestion 
btn_deleteQuestion_{{sortOrder}}